### ISSUE TITLE
ModuleNotFoundError: No module named arepl_dump

### ISSUE THREAD

**Issue by eliassal** [Almenon/AREPL-vscode#379]
**Describe the bug**
I copied the code for Misc/Dumping, once pasted, VSC puts a red suiggly line under "arepl_dump" in
`from arepl_dump import dump`
I hit run without debug, I get variables dumped on the right hand side but in terminal I get 

> File "c:\Projects\Python\10 Py Apps10Days\dumpallvar.py", line 1, in <module>
>     from arepl_dump import dump
> ModuleNotFoundError: No module named 'arepl_dump'

**To Reproduce**
1. Described above

**Expected behavior**
it should work according to what is mentioned at https://marketplace.visualstudio.com/items?itemName=almenon.arepl

**Screenshots**
![AREPL-VSC](https://user-images.githubusercontent.com/8447864/112466885-a3d8b100-8d66-11eb-9464-29f692b60852.jpg)

**Other Information (please complete the following information):**
 - OS: Windows10 64bit
 - Python Version 3.9

**Comment by Almenon** [comment]
arepl_dump is not meant to be run from the command line - it only works in AREPL.

**Comment by Almenon** [comment]
That being said it would probably still be good to have it not cause an error when running from the command line, so users can run a script from the shell without having to comment the import out 🤔

**Comment by eliassal** [comment]
Yes, good idea is not to generate the suiglly red line and leave us run commands without errors. Can this be fixed?

**Comment by Almenon** [comment]
It can, but I don't work that much on this project anymore. You or someone else is welcome to be submit a PR if you wish.

**Comment by vbrozik** [comment]
It would be great to get rid of the reported problem. For example if Pylance is being used (I think it is the default in VS Code) it shows the red underline and this message in the problems report:

> Import "arepl_dump" could not be resolved

Of course constantly reported problems make the otherwise extremely useful problem report hard to use.

For Pylance a simple workaround could be to ignore the problem by appending a directive:
``` Python
from arepl_dump import dump     # type: ignore
```

**Comment by eliassal** [comment]
@vbrozik , not sure what you suggest resolves the issue. I have the directive but still getting 

ModuleNotFoundError: No module named 'arepl_dump',  as you can notice in the snapshot

![arepl-Dump](https://user-images.githubusercontent.com/8447864/144222797-94eea296-5fab-4596-a729-7088aeeb402b.jpg)

**Comment by vbrozik** [comment]
@eliassal, I was reacting just to:
> good idea is not to generate the suiglly red line

Of course the directive just deactivates the Pyright/Pylance check (inside VS Code) to get rid of the red line. It will not make the code run outside of AREPL.

**Comment by bernhardkaindl** [comment]
I've had this error even inside vscode, at least when using a venv. For this and the case when run outside of vscode, this  ìmport` appears to help (yes, the `finally`seems to be needed as well, otherwise I get a new editor with these lines in it as an Unitled file an of course arepl does not work then:
```py
try:
    from arepl_dump import dump  # type: ignore
except ModuleNotFoundError:
    def dump():
        pass
finally:
    pass
```
